easybuild/easyconfigs/p/PennCNV/PennCNV-Affy-2009Aug27.eb

```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Jonas Demeulemeester
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'PennCNV-affy'
```

Comment:
@jdemeul The naming here needs to be consistent with the file name, so both should be `Affy` or `affy`.